tests/cases/standalone/catalog/schema.sql

```diff
@@ -1,27 +1,31 @@
-CREATE SCHEMA test;
+CREATE SCHEMA test_schema;
 
 SHOW DATABASES;
 
-CREATE TABLE test.hello(i BIGINT TIME INDEX);
+CREATE TABLE test_schema.hello(i BIGINT TIME INDEX);
 
-DROP TABLE test.hello;
+DROP TABLE test_schema.hello;
 
-DROP SCHEMA test;
+DROP SCHEMA test_schema;
```

Comment:
I can find methods like `deregister_schema` in the catalog manager. There is only a `register_schema`. Looks like we should also implement deregister_schema?

Reply:
`drop schema` is tracked in issue https://github.com/GreptimeTeam/greptimedb/issues/825